README.md

```diff
@@ -1,9 +1,11 @@
-# go-execplus
+# go-execplus  [![Build Status](https://travis-ci.org/Originate/go-execplus.svg?branch=master)](https://travis-ci.org/Originate/go-execplus)
```

Comment:
Done. That's an excellent idea, we should change all our repos to something like that. I personally find the "builds" tab more useful, and it also loads faster.